### PR TITLE
View search

### PR DESCRIPTION
**Pull request by Bouncyyahomie**
Done search bar but also has some bug with navbar

### CODE REVIEW

**Comment by Chayapol-c** [comment]
I think it needs to fix tests because Travis CI build still failed.

File: appointment/views.py [hunk 1]
```diff
@@ -45,13 +51,27 @@ def get_context_data(self, **kwargs):
         context['next_month'] = next_month(this_day)
         return context
 
-def meeting_list(request, year,month,day):
-    meetings = Meeting.objects.filter(start_time__year=year,start_time__month=month, start_time__day=day).order_by('start_time')
-    context = {'meeting': meetings} 
+
+def meeting_list(request, year, month, day):
```
Comment (Chayapol-c) [inline appointment/views.py:55]: format code

File: appointment/views.py [hunk 2]
```diff
@@ -45,13 +51,27 @@ def get_context_data(self, **kwargs):
...
+    meetings = Meeting.objects.filter(start_time__year=year, start_time__month=month, start_time__day=day).order_by(
+        'start_time')
+    context = {'meeting': meetings}
     return render(request, 'appointment/meeting_list.html', context)
 
-def detail(request,meeting_id):
+
+def detail(request, meeting_id):
```
Comment (Chayapol-c) [inline appointment/views.py:62]: format code

File: requirements.txt [hunk 3]
```diff
@@ -1,5 +1,5 @@
 coverage
-Django
+Django~=3.1.1
```
Comment (Chayapol-c) [inline requirements.txt:2]: add Django version to requirements.txt

File: appointment/views.py [hunk 4]
```diff
@@ -1,33 +1,39 @@
 """Views for Django appointment app."""
-from django.shortcuts import render , get_object_or_404
+from django.shortcuts import render, get_object_or_404
 from django.views import generic
+
```
Comment (Chayapol-c) [inline appointment/views.py:4]: blank lines below are formated

File: appointment/urls.py [hunk 5]
```diff
@@ -9,4 +10,5 @@
     path('', views.IndexView.as_view(), name="home_page"),
     path("meeting/<int:year>/<int:month>/<int:day>", views.meeting_list, name="meet_list"),
     path("<int:meeting_id>/detail", views.detail, name="detail"),
+    url(r'result/$', views.search, name="search"),
```
Comment (Chayapol-c) [inline appointment/urls.py]: I think it should use path() instead of url().

**Review by nuttapol-kor — APPROVED** [review]
it's ok update for some search bar. Build Passed!!

**Review by Chayapol-c — APPROVED** [review]
Some changes in base.html are for aligning nav-bar.

**Pull request merged**